addons/test_mail/tests/test_performance.py

```diff
@@ -1875,7 +1875,7 @@ def test_message_post_loop(self):
         self.push_to_end_point_mocked.reset_mock()  # reset as executed twice
         self.flush_tracking()
 
-        with self.assertQueryCount(employee=800):  # tm: 791
+        with self.assertQueryCount(employee=809):  # tm: 800
```

Comment:
Note: this query count is incremented due to the addition of the `thumbnail` field on `ir.attachment`